pkg/web/templates/partials/transaction/transaction_payload.html

```diff
@@ -1 +1,32 @@
-<div></div>
\ No newline at end of file
+<div class="modal-box">
+  <div class="flex justify-between items-center">
+    <h1 class="font-bold text-xl">Complete Transaction</h1>
+    <button id="transaction-complete-close-btn" onclick="transaction_complete_modal.close()" class="btn btn-sm btn-circle btn-ghost">
+      <i class="fa-solid fa-x"></i>
+      <span class="sr-only">Close modal</span>
+    </button>
+  </div>
+  <p class="py-4">Send the on-chain transaction ID to the counterparty so that they may identify the transaction record.</p>
+  <div>
+    <form
+      id="transaction-complete-form"
+      hx-post="/v1/transactions/{{ .EnvelopeID }}/send"
+      hx-ext="json-enc"
+      hx-swap="none"
+      hx-indicator="#loader"
+      method="dialog"
+      >
+      <input id="envelope" name="envelope" type="hidden" value="{{ .EnvelopeDump }}" />
+      <div class="mb-4">
+        <label for="txid" class="label-style pb-0.5">Transaction ID (Txid)</label>
```

Comment:
The `send envelope` form label was updated to now display `Transaction Hash`. Should this be changed to `Transaction Hash` or  `Transaction ID / Hash`?

Reply:
I like `Transaction ID / Hash` -- the issue is that there are multiple chains that this can be used with, so trying to stay as generic as possible is important.